lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -29,7 +29,14 @@ define([
                             modified: '2016-03-25T13:45:21.220732Z',
                             marketing_slug: 'p_2?param=haha&test=b', 
                             id: 146,
-                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b'
+                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b',
+                            /* jshint ignore:start */
+                            banner_image_urls: [
+                                '348x116': 'http://www.edx.org/images/org1/test1',
+                                '435x145': 'http://www.edx.org/images/org1/test2',
+                                '726x242': 'http://www.edx.org/images/org1/test3'
+                            ]
+                            /* jshint ignore:end */
```

Comment:
@jimabramson I think prepending an 's' would be fine. I don't think it's just a JSHint thing 'tho, it's the way JS handles numbers and I would like to be cautious in that regard when adding future keys to objects to make sure they don't start with numbers, or do any other things that JS could potentially balk at.